### PR TITLE
Add noir

### PR DESCRIPTION
**Pull request by mmatongo**

### CODE REVIEW

**Comment by AMDmi3** [comment]
We can't support this due to the same reason we no longer support kiss, see #1166.

**Comment by AMDmi3** [comment]
We can't support this due to the same reason we no longer support kiss, see #1166.

**Comment by mmatongo** [comment]
> We can't support this due to the same reason we no longer support kiss, see #1166.

I noticed that but we won't be using the new kiss variables for source files, instead we will maintain the old structure.

Will this have any affect on support?

**Comment by AMDmi3** [comment]
If you don't use placeholders it should be OK. You currently do use them: https://github.com/noirlinux/main/blob/master/core/zlib/sources

**Comment by mmatongo** [comment]
> If you don't use placeholders it should be OK. You currently do use them: https://github.com/noirlinux/main/blob/master/core/zlib/sources

They will be removed within the next couple of hours.

**Comment by mmatongo** [comment]
I'm working on it right now.

**Comment by AMDmi3** [comment]
Note that I don't in fact want to enforce upstream repository format in any way, I'm just not going to implement complex parsing. Alternatively, you could leave them (I see these as pretty convenient) be and provide a preprocessed package information dump for repology.

**Comment by mmatongo** [comment]
I can go either way, but in this case to avoid being impeded I will most likely revert to the old format to avoid unnecessary complexities.

File: repos.d/kiss/noir.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+###########################################################################
+# NOIR Linux
+###########################################################################
+- name: noir
+  type: repository
+  desc: NOIR Linux Main
+  family: noir
```
Comment (AMDmi3) [inline repos.d/kiss/noir.yaml:7]: You also need to list this in [repology-schemacheck.py](repology-schemacheck.py)

Comment (mmatongo) [reply repos.d/kiss/noir.yaml:7]: done.

**Comment by mmatongo** [comment]
> If you don't use placeholders it should be OK. You currently do use them: https://github.com/noirlinux/main/blob/master/core/zlib/sources

This has been cleared up now.

**Pull request merged**